package.json

```diff
@@ -2,14 +2,18 @@
   "name": "kolibri-explore-plugin",
   "description": "Monorepo with kolibri's Discovery page and custom channel presentations.",
   "private": true,
+  "config": {
+    "kolibri_tools_i18n_modules": "--plugins kolibri_explore_plugin --namespace ek-components --searchPath ./packages/ek-components --namespace template-ui --searchPath ./packages/template-ui --namespace welcome-screen --searchPath ./packages/welcome-screen"
+  },
   "scripts": {
-    "build": "yarn build:version && yarn build:packages && yarn deploy:welcome && yarn build:plugin && yarn build:info",
+    "build": "yarn build:version && yarn build:packages && yarn deploy:welcome && yarn build:plugin && yarn build:info && yarn build:i18n",
     "build:plugin": "./scripts/set_override.py default && kolibri-tools build prod --plugins kolibri_explore_plugin --transpile",
     "build:packages": "yarn build:libs",
     "build:libs": "lerna run build --ignore template-ui",
     "build:apps": "lerna run build --scope template-ui",
     "build:info": "./scripts/build_info.sh > kolibri_explore_plugin/static/build-info.json",
     "build:version": "python -m setuptools_scm",
+    "build:i18n": "yarn run i18n-extract && yarn run i18n-django-compilemessages",
     "build-dist": "./scripts/clean.sh && yarn clean && yarn build && python -m build",
```

Comment:
Am I wrong or does `i18n-download-translations` need to be run by one of these build targets to make sure the compiled messages are included in the package? Or is the idea that the compiled messages are committed and therefore we want to release that instead of regenerating them at the release time?

Reply:
Following what Kolibri do, the compiled messages (`.mo`, `kolibri_explore_plugin/assets/src/{vue-,}intl-locale-data.js`) are committed, so you’re right that the idea is that we release those rather than regenerating them.